pkg/spec/populators_test.go

```diff
@@ -21,6 +21,9 @@ import (
 	"sort"
 	"testing"
 
+	"fmt"
```

Comment:
can you arrange the imports similar to the comment here https://github.com/kedgeproject/kedge/pull/149#issuecomment-315051859